contracts/epochs/FlowEpoch.cdc

```diff
@@ -674,24 +674,36 @@ pub contract FlowEpoch {
 
     /// Borrow a reference to the FlowIDTableStaking Admin resource
     access(contract) fun borrowStakingAdmin(): &FlowIDTableStaking.Admin {
-        let adminRef = self.account.borrow<&FlowIDTableStaking.Admin>(from: FlowIDTableStaking.StakingAdminStoragePath)
-            ?? panic("Could not borrow staking admin")
+        let adminCapability = self.account.copy<Capability>(from: FlowIDTableStaking.StakingAdminStoragePath)
+            ?? panic("Could not get capability from account storage")
+
+        // borrow a reference to the staking admin object
+        let adminRef = adminCapability.borrow<&FlowIDTableStaking.Admin{FlowIDTableStaking.EpochOperations}>()
+            ?? panic("Could not borrow reference to staking admin")
 
         return adminRef
     }
 
     /// Borrow a reference to the ClusterQCs Admin resource
     access(contract) fun borrowClusterQCAdmin(): &FlowClusterQC.Admin {
-        let adminRef = self.account.borrow<&FlowClusterQC.Admin>(from: FlowClusterQC.AdminStoragePath)
-            ?? panic("Could not borrow qc admin")
+        let adminCapability = self.account.copy<Capability>(from: FlowClusterQC.AdminStoragePath)
```

Comment:
Just wondering - why is it using `copy` instead of `getCapability`?

Reply:
`getCapability` only works for getting a capability directly from a path that has a link, which can only be private or public paths. Since this is a capability from another account, we have to copy it from storage